### PR TITLE
chore(credentials): Update design proposal

### PR DESCRIPTION
**Pull request by zregvart**
Updates the design proposal by removing the technical details not needed or not realized, updates the REST endpoints and the example payloads.

### CODE REVIEW

File: proposals/003-credentials.md [hunk 1]
```diff
@@ -24,113 +24,30 @@ My requirements are:
...
 
-Management of user and system credentials is also performed by credential provider, for instance initial setup of system credentials or refresh of user credentials.
+Management of user and system credentials is also performed by credential provider, for instance initial setup of system credentials or refresh of user credentials (where applicable).
 
 
 ## User interaction
 
-A citizen user during the creation of new connection selects `conect to _Service_`, this leads the user to the 3rd party service requiring the users credential to authorise Syndesis access on the users behalf. The user authenticates and authorizes Syndesis's access, and is returned to the same create connection screen which now shows that Syndesis is allowed to access the service.
+A citizen user during the creation of new connection selects `conect to _Service_`, this leads the user to the 3rd party service requiring the users credential to authorize Syndesis access on the users behalf. The user authenticates and authorizes Syndesis's access, and is returned to the same create connection screen which now shows that Syndesis is allowed to access the service.
```
Comment (jimmidyson) [inline proposals/003-credentials.md]: typo: `conect` -> `connect`

Comment (jimmidyson) [reply proposals/003-credentials.md]: btw `authorize` and `authorise` are both fine in the land of real English :stuck_out_tongue:

**Review by jimmidyson — CHANGES_REQUESTED** [review]
one typo, but otherwise LGTM

File: proposals/003-credentials.md (hunk 1 above)
Comment (rhuss) [inline proposals/003-credentials.md]: nit: "connect to _Service_"

**Comment by rhuss** [comment]
thanks !

**Pull request merged**